examples/plot_simple_experiment.py

```diff
@@ -36,7 +38,13 @@
 # make the stimuli if necessary and then load them
 fname = 'equally_spaced_sinewaves.hdf5'
 if not op.isfile(fname):
-    from stimuli.plot_generate_simple_stimuli import generate_stimuli
+    # This sys.path wrangling is only necessary for Sphinx automatic
+    # documentation building
+    sys.path.insert(0, os.getcwd())
+    try:
+        from plot_generate_simple_stimuli import generate_stimuli
+    finally:
+        sys.path.pop(0)
```

Comment:
I have to say, I like sphinx-doc less and less every day.  aside from the unintuitive file names (`plot_whatever.py` for files that don't involve plotting) and the accompanying calls `screenshot`, now we have this.  I worry that it does more to obfuscate these tutorial examples than it does to improve them...